### PR TITLE
Added Cisco APs & Controllers + Rugged Switches

### PR DESCRIPTION
**Pull request by rc-networks**
Added 'Cisco': 
AIR-AP2802I-Q-K9
C9120AXI-Q
C9136I-Q
C9800-CL-K9
AIR-AP1562I-Q-K9
IE-3400-8P2S
C9130AXI-Q

### CODE REVIEW

File: device-types/Cisco/AIR-AP2802I-Q-K9.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+part_number: AIR-AP2802I-Q-K9
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Aironet 2800 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-2800-series-access-points/datasheet-c78-736497.html)'
+interfaces:
+  - name: WAN
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
+    type: ieee802.11ac
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP2802I-Q-K9.yaml:14]: Same comment about interface naming and PoE spec

File: device-types/Cisco/C9120AXI-Q.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+manufacturer: Cisco
+model: Catalyst 9120AXI-Q
+slug: cisco-c9120axi-Q
+part_number: C9120AXI-Q
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Catalyst 9120AX Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9120ax-series-access-points/datasheet-c78-742115.html)'
+interfaces:
+  - name: WiFi
+    type: ieee802.11ax
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXI-Q.yaml:11]: Config interface names on the 9130:

wired0
wifi0
wifi1

"Physical" Interface Names:

GigabitEthernet0
Dot11Radio0
Dot11Radio1

File: device-types/Cisco/C9130AXI-Q.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+part_number: C9130AXI-Q
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Catalyst 9130AX Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9100ax-access-points/nb-06-cat-9130-ser-ap-ds-cte-en.html)'
+interfaces:
+  - name: WiFi
+    type: ieee802.11ax
+  - name: GigabitEthernet0
+    type: 5gbase-t
+    poe_mode: pd
```
Comment (DanSheps) [inline device-types/Cisco/C9130AXI-Q.yaml:14]: Same comment about naming

File: device-types/Cisco/C9136I-Q.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+part_number: C9136I-Q
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Catalyst 9136 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9100ax-access-points/nb-06-cat9136-access-point-ds-cte-en.html)'
+interfaces:
+  - name: WiFi
+    type: ieee802.11ax
+  - name: GigabitEthernet0
+    type: 5gbase-t
+    poe_mode: pd
```
Comment (DanSheps) [inline device-types/Cisco/C9136I-Q.yaml:14]: Same comment about naming

File: device-types/Cisco/C9800-CL-K9.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: Cisco
+model: Catalyst 9800-CL-K9
+slug: cisco-c9800-cl-k9
+part_number: C9800-CL-K9
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 9800-CL Wireless Controller for Cloud Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9800-cl-wireless-controller-cloud/nb-06-cat9800-cl-cloud-wirel-data-sheet-ctp-en.html)'
```
Comment (DanSheps) [inline device-types/Cisco/C9800-CL-K9.yaml:8]: This is really a VM, not a device, it should not be included.

File: device-types/Cisco/IE-3400-8P2S.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+manufacturer: Cisco
+model: IE-3400-8P2S
+slug: cisco-ie-3400-8p2s
+part_number: IE-3400-8P2S
+# comments: requires seperate PSU which is then wired into DC-A DC-B screw terminals
+u_height: 2
```
Comment (DanSheps) [inline device-types/Cisco/IE-3400-8P2S.yaml:7]: Weight is available on the datasheet, please include weight

File: device-types/Cisco/C9136I-Q.yaml [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: Cisco
+model: Catalyst 9136I-Q
+slug: cisco-c9136i-q
+part_number: C9136I-Q
+u_height: 0
+is_full_depth: false
```
Comment (DanSheps) [inline device-types/Cisco/C9136I-Q.yaml:7]: Weight is available, please include weight

File: device-types/Cisco/C9130AXI-Q.yaml [hunk 8]
```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: Cisco
+model: Catalyst 9130AXI-Q
+slug: cisco-c9130axi-q
+part_number: C9130AXI-Q
+u_height: 0
+is_full_depth: false
```
Comment (DanSheps) [inline device-types/Cisco/C9130AXI-Q.yaml:7]: Weight is available, please include weight

File: device-types/Cisco/C9120AXI-Q.yaml [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: Cisco
+model: Catalyst 9120AXI-Q
+slug: cisco-c9120axi-Q
+part_number: C9120AXI-Q
+u_height: 0
+is_full_depth: false
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXI-Q.yaml:7]: Weight is available, please include weight

File: device-types/Cisco/AIR-AP2802I-Q-K9.yaml [hunk 10]
```diff
@@ -0,0 +1,14 @@
+---
+manufacturer: Cisco
+model: Aironet 2802I-Q
+slug: cisco-air-ap2802i-q-k9
+part_number: AIR-AP2802I-Q-K9
+u_height: 0
+is_full_depth: false
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP2802I-Q-K9.yaml:7]: Weight is available, please include weight

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml [hunk 11]
```diff
@@ -0,0 +1,17 @@
+---
+manufacturer: Cisco
+model: Aironet 1562I-Q-K9
+slug: cisco-air-ap1562i-q-k9
+part_number: AIR-AP1562I-Q-K9
+u_height: 0
+is_full_depth: false
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1562I-Q-K9.yaml:7]: Weight is available, please include weight

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+comments: '[Cisco Aironet 1560 Series Outdoor Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1560-series/datasheet-c78-737416.html)'
+interfaces:
+  - name: WAN
```
[the remark this answers is not in the document]
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml:10]: hmm, That means all the `Aironet`s that are currently on the master branch are wrong.
Checked with DNA-C, it's named `GigabitEthernet0`
Will be updated on the next commit.

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+    type: 1000base-t
+    poe_mode: pd
```
[the remark this answers is not in the document]
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml:12]: I believe this is correct, here is the [link](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1560-series/datasheet-c78-737416.html#:~:text=WAN%20port%2010/100/1000BASE%2DT%20Ethernet%2C%20autosensing%20(RJ%2D45)%2C%20PoE%20in).

Or is there anything I need to add, this is the same as the other `Aironet`s that are available in the master branch

[the remark this answers is not in the document]
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml:12]: If you mean the `poe_type` I'll add it on the next commit

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+  - name: WiFi
```
[the remark this answers is not in the document]
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml:13]: Ok, I checked DNAC and our WLCs looking for the Wireless interface name and I can't seem to find them.
Where do you think this information resides? 

By the way, this `WiFi` interface name is the same as the ones across the `Aironet` series that are on the master branch right now.

File: device-types/Cisco/AIR-AP2802I-Q-K9.yaml (hunk 1 above)
Comment (rc-networks) [reply device-types/Cisco/AIR-AP2802I-Q-K9.yaml:14]: Where do I find the WLAN interface names?
These names were based on other available `Aironet` device_types in the master branch

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml (hunk 11 above)
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml:7]: Will be added on next commit

File: device-types/Cisco/AIR-AP2802I-Q-K9.yaml (hunk 10 above)
Comment (rc-networks) [reply device-types/Cisco/AIR-AP2802I-Q-K9.yaml:7]: Will be added on next commit

File: device-types/Cisco/C9120AXI-Q.yaml (hunk 2 above)
Comment (rc-networks) [reply device-types/Cisco/C9120AXI-Q.yaml:11]: This is a C9120, Please, do advice where can we find these config interface names.
I only got something like:
image

and nothing close to:
```
wired0
wifi0
wifi1
```

File: device-types/Cisco/AIR-AP1562I-Q-K9.yaml (hunk 14 above)
[the remark this answers is not in the document]
Comment (rc-networks) [reply device-types/Cisco/AIR-AP1562I-Q-K9.yaml]: Removed Wifi and added Radio Interfaces instead

File: device-types/Cisco/AIR-AP2802I-Q-K9.yaml (hunk 1 above)
Comment (rc-networks) [reply device-types/Cisco/AIR-AP2802I-Q-K9.yaml]: Interface names, fixed.

File: device-types/Cisco/C9120AXI-Q.yaml (hunk 2 above)
Comment (rc-networks) [reply device-types/Cisco/C9120AXI-Q.yaml]: Fixed interface names based on DNAC and WLC Radio interface outputs.

File: device-types/Cisco/C9800-CL-K9.yaml (hunk 5 above)
Comment (rc-networks) [reply device-types/Cisco/C9800-CL-K9.yaml]: removed

**Review by danner26 — CHANGES_REQUESTED** [review]
The `IE-3400-8P2S` is missing a power definition. It appears it might be `dc-terminal`s?

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Pull request closed without merging**